core/server/i18n.js

```diff
@@ -23,8 +36,14 @@ I18n = {
      * @returns {string}
      */
     t: function t(path, bindings) {
-        var string = I18n.findString(path),
-            msg;
+        var string, defaultString, msg;
+
+        currentLocale = settingsCache.get('default_locale') || 'en';
+        if (bindings !== undefined) {
+            defaultString = bindings.defaultString;
+            delete bindings.defaultString;
```

Comment:
Where is that used?

Reply:
That was implemented on May 31, following Ghost team's really great suggestion of falling back to the default English text inside {{t}} translation helpers, in case of missing translations. It has worked really well.